Update Hosting documentation (#8510)
Scille/parsec-cloud: docs/hosting/deployment/index.rst

.. cspell:words conninfo literalinclude linenos

.. _doc_hosting_deployment:

.. role:: bash(code)
  :language: bash

.. role:: yaml(code)
  :language: yaml

=================
Server Deployment
=================

> This section covers how to deploy the Parsec server .

Reviewer: ```suggestion
This section covers how to deploy the Parsec server.
```

> This guide covers the deployment procedure for the `Parsec`_ server.

Reviewer: I'm not sure the link to https://parsec.cloud is that useful here
```suggestion
This section covers how to deploy the Parsec server .
```
